mediorum/server/serve_health.go

```diff
@@ -60,6 +61,11 @@ type healthCheckResponseData struct {
 func (ss *MediorumServer) serveHealthCheck(c echo.Context) error {
 	healthy := ss.databaseSize > 0
 
+	allowUnregistered, _ := strconv.ParseBool(c.QueryParam("allow_unregistered"))
```

Comment:
is the queryparam gonna default to allow_unregistered?

Reply:
no this ParseBool function will default to false so by default it will return 506 if you don't include the queryparam with an `=true` or similar

Reply:
how does the default work?

Reply:
if the queryparam isn't passed then it'll just be an empty string, which will make the parsing return false for `allowUnregistered` [here](https://cs.opensource.google/go/go/+/refs/tags/go1.20.7:src/strconv/atob.go;l=17)

Reply:
oh my b i was hung up on the documentation

> It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error.

sooo it still errors and returns false. maybe that should be handled but either way is ok